dashboard/test/controllers/lti_v1_controller_test.rb

```diff
@@ -0,0 +1,59 @@
+require "test_helper"
+
+class LtiV1ControllerTest < ActionDispatch::IntegrationTest
+  setup_all do
+    @integration = create :lti_integration
+  end
+
+  test 'given no params, return unauthorized' do
+    get '/lti/v1/login', params: {}
+    assert_response :unauthorized
+  end
+
+  test 'given a client_id that doesn not exist, return unauthorized' do
+    get '/lti/v1/login', params: {client_id: 'nope', iss: @integration.issuer}
```

Comment:
This requires knowing that 'nope' is not the factory's client_id. Would an empty string also work?

Reply:
Good call, I'll change this to an empty string.